java/arcs/android/devtools/devtools_message.proto

```diff
@@ -16,3 +17,12 @@ message DevToolsProxyMessageProto {
   string storage_key = 2;
   StoreType store_type = 3;
 }
+
+message DevToolsInitProto {
+  int32 dev_tools_version = 1;
+  enum Platform {
+    ANDROID = 0;
```

Comment:
Best practices are to make 0 a PLATFORM_UNKNOWN value, and start your real values from 1.